image.yaml

```diff
@@ -110,14 +112,15 @@ modules:
           - name: os-java-jolokia
           - name: jolokia
           - name: os-eap7-openshift
-          - name: os-eap71-openshift
+          - name: os-eap-online-openshift
           - name: os-eap7-modules
-          - name: os-eap71-modules
+          - name: os-eap-online-modules
```

Comment:
Were we planning on housing these in jboss-eap-modules instead of cct_module?  Also, could we use package naming, maybe jboss.eap.openshift.launch and jboss.eap.openshift.modules?  Do they need to be separate modules?